docs/contributors/maintainers.md

```diff
@@ -125,7 +125,14 @@ Create a new GitHub release by following the [GitHub help](https://help.github.c
   available in the branch
   - in that case, choose a dedicated commit 
   - **important**: ensure that the build is passing on the chosen branch or commit used to create the release
+- Description
+  - add a link to the bpmn-visualization related release
+  - add a link to the live environment for the related tag
+  - make GitHub generates the [release notes automatically](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes)
 
+You should have something like in release [v0.21.2](https://github.com/process-analytics/bpmn-visualization-examples/releases/tag/v0.21.2)
+
+![release process automation overview](./images/release_process_part-10_examples_release-notes.png)
```

Comment:
❓ I guess the screenshot could be better (it currently misses the left and bottom border) 😒

Reply:
I agree for the screenshot 🙂

Reply:
Screenshot updated in 4fbf12a